controllers/api/base.go

```diff
@@ -40,23 +42,58 @@ func (c *BaseController) Prepare() {
 	c.handleAuthorizeRequest()
 }
 
+func (c *BaseController) GetPageSize() (pageSize int) {
+	return DefaultPageSize
+}
+
+func (c *BaseController) GetOrderBy() (orderBy []string) {
+	return DefaultOrderBy
+}
+
 func (c *BaseController) MappingPolicy(method string, policy Policy) {
 	c.actionPolicy[method] = policy
 }
 
-func (c *BaseController) RenderJSON(data interface{}, status int) {
-	c.Ctx.Output.Header("Content-Type", ContentType)
-	c.Ctx.ResponseWriter.WriteHeader(status)
+func (c *BaseController) RenderJSONMany(data interface{}, meta *jsonapi.Meta, links *jsonapi.Links, status int) {
```

Comment:
`RenderJSONMany` name is odd to me 🤔

Reply:
@junan I make it consistences with this https://github.com/google/jsonapi/blob/master/node.go#L25
But I'll change to the other name